polybar-scripts/info-timew/README.md

```diff
@@ -0,0 +1,13 @@
+# Script: timew
+
+This script displays the daily tracked time. You can click it, to start tracking, using timew.
+
+## Module
+
+```ini
+[module/timew]
+type = custom/script
+exec = ~/.config/polybar/scripts/timew.sh
+interval = 2
```

Comment:
A small interval like this is always a bad idea. A script with the `tail` option would fit better. But it would probbaly things make to complicated here.